internal/ingress/controller/store/store.go

```diff
@@ -773,3 +837,58 @@ func (s k8sStore) Run(stopCh chan struct{}) {
 		go wait.Until(s.checkSSLChainIssues, 60*time.Second, stopCh)
 	}
 }
+
+// getCurrentPod returns the current Pod based on environment namespace and name
+func (s k8sStore) getCurrentPod() *corev1.Pod {
+	ns := os.Getenv("POD_NAMESPACE")
```

Comment:
@aledbf do you know any better of getting current controller `Pod`?

Reply:
I've replaced this with `k8s.GetPodDetails` https://github.com/kubernetes/ingress-nginx/pull/3455/files#diff-ece58b3bebec02a6d3173c8d8099d298R257